api/src/opentrons/protocol_engine/state/pipettes.py

```diff
@@ -85,6 +85,14 @@ class BoundingNozzlesOffsets:
     front_right_offset: Point
 
 
+@dataclass(frozen=True)
+class PipetteBoundingBoxOffsets:
+    """Offsets of the bounding nozzles of the pipette."""
```

Comment:
```suggestion
    """Offsets of the bounding box of the pipette."""
```

Reply:
Hmm, they are offsets tho. How about-
```suggestion
    """Offsets of the corners of the pipette's bounding box."""
```
?